app/styles/play/level/control-bar-view.sass

```diff
@@ -172,13 +172,22 @@
         .hamburger span.icon-bar
           background-color: $control-yellow-highlight
 
-    #level-done-button, #next-game-button, #control-bar-sign-up-button
+    #next-game-button, #control-bar-sign-up-button
       top: 7px
       font-size: 13px
       height: 24px
 
     #level-done-button
-      display: none
```

Comment:
The level done button that was removed was previously styled hidden, so I assumed that it wasn't being used anywhere, can this style somehow be changed dynamically on runtime? signifying a possibility of usage of the done button previously?

Reply:
hmmm I found this line at PlayLevelView.js line 1112
` $('#level-done-button').show()`
so the previous level done button might be used after all